crates/bevy_window/src/window.rs

```diff
@@ -428,14 +444,19 @@ impl Default for Cursor {
 )]
 #[reflect(Debug, PartialEq)]
 pub enum WindowPosition {
-    /// Position will be set by the window manager
+    /// Position will be set by the window manager.
+    /// Bevy will delegate this decision to it and no guarantees can be made about where the window will be placed.
```

Comment:
"it" is confusing here

```suggestion
    /// Bevy will delegate this decision to the window manager and no guarantees can be made about where the window will be placed.
```